src/components/RightSidebar/Matterbridge/MatterbridgeSettings.vue

```diff
@@ -20,59 +20,79 @@
 -->
 
 <template>
-	<div>
+	<div class="matterbridge-settings">
 		<div v-if="loading" class="loading" />
 		<div v-show="!loading">
+			<h3>
+				<span class="icon icon-category-integration" />
+				<p>{{ t('spreed', 'Bridge with other services') }}</p>
+			</h3>
 			<div id="matterbridge-header">
-				<h3>
-					{{ t('spreed', 'Bridge with other services') }}
-				</h3>
 				<p>
 					{{ t('spreed', 'You can bridge channels from various instant messaging systems with Matterbridge.') }}
 					<a href="https://github.com/42wim/matterbridge/wiki" target="_blank" rel="noopener">
-						{{ t('spreed', 'More info on Matterbridge.') }}
+						<span class="icon icon-external" />
+						{{ t('spreed', 'More info on Matterbridge') }}
 					</a>
 				</p>
 			</div>
 			<div class="basic-settings">
-				<ActionCheckbox
-					:token="token"
-					:checked="enabled"
-					@update:checked="onEnabled">
-					{{ t('spreed', 'Enabled') }}
-					({{ processStateText }})
-				</ActionCheckbox>
-				<button class="" @click="showLogContent">
-					{{ t('spreed', 'Show matterbridge log') }}
-				</button>
-				<Modal v-if="logModal"
-					@close="closeLogModal">
-					<div class="modal__content">
-						<textarea v-model="processLog" class="log-content" />
-					</div>
-				</Modal>
-				<Multiselect
-					ref="partMultiselect"
-					v-model="selectedType"
-					label="displayName"
-					track-by="type"
-					:placeholder="newPartPlaceholder"
-					:options="formatedTypes"
-					:internal-search="true"
-					@input="clickAddPart" />
-				<ActionButton
-					icon="icon-checkmark"
-					@click="onSave">
-					{{ t('spreed', 'Save') }}
-				</ActionButton>
+				<div v-show="!enabled"
+					class="add-part-wrapper">
+					<span class="icon icon-add" />
+					<Multiselect
+						ref="partMultiselect"
+						v-model="selectedType"
+						label="displayName"
+						track-by="type"
+						:placeholder="newPartPlaceholder"
+						:options="formatedTypes"
+						:user-select="true"
+						:internal-search="true"
+						@input="clickAddPart">
+						<template #option="{option}">
+							<img class="icon-multiselect-service"
+								:src="option.iconUrl">
+							{{ option.displayName }}
+						</template>
+					</Multiselect>
```

Comment:
This breaks line with Dyslexia font from the accessibility settings:
![Bildschirmfoto von 2020-12-15 13-38-49](https://user-images.githubusercontent.com/213943/102216091-f0e0f780-3eda-11eb-808d-f6ef30873e46.png)